contribs/gmf/src/options.js

```diff
@@ -48,28 +48,10 @@ import {createDefaultStyle} from 'ol/style/Style.js';
  * radius creation tool. The value is in map units.
  */
 
-/**
- * Specify radius for regular polygons, or radius1 and radius2 for stars.
- * See also: https://openlayers.org/en/latest/examples/regularshape.html
- *
- * @typedef {Object} RegularShapeOptions
- * @property {import("ol/style/Fill.js").Options} [fill] Fill style.
- * @property {number} points Number of points for stars and regular polygons. In case of a polygon, the number of points
- * is the number of sides.
- * @property {number} [radius] Radius of a regular polygon.
- * @property {number} [radius1] Outer radius of a star.
- * @property {number} [radius2] Inner radius of a star.
- * @property {number} [angle=0] Shape's angle in degree. A value of 0 will have one of the shape's point facing up.
- * @property {Array<number>} [displacement=[0,0]] Displacement of the shape
- * @property {import("ol/style/Stroke.js").Options} [stroke] Stroke style.
- * @property {number} [rotation=0] Rotation in degree (positive rotation clockwise).
- * @property {boolean} [rotateWithView=false] Whether to rotate the shape with the view.
- */
-
 /**
  * Configuration options for the permalink service.
- * @typedef {Object} gmfPermalinkOptions
- * @property {RegularShapeOptions[]} [crosshairStyle] An alternate
+ * @typedef {object} gmfPermalinkOptions
+ * @property {StyleLike} [crosshairStyle] An alternate
```

Comment:
I think you can write this comment on a single row

Reply:
No, it just misses a little...